pkg/kubecost/allocation.go

```diff
@@ -246,16 +246,62 @@ func (pva *PVAllocation) Equal(that *PVAllocation) bool {
 		util.IsApproximately(pva.Cost, that.Cost)
 }
 
+// used to compute the average of 2 PARCs. A PARC is a dimensonless
+// percentage that loses information needed when finding an average
+// to that end, we track the percentage and resource usage of the
+// component used to make the parc, to allow us to average two PARCs
+type ParcsComponent struct {
+	TotalCost       float64
+	UsageProportion float64
+}
+
+func (p *ParcsComponent) Clone() ParcsComponent {
```

Comment:
Just a note--it's a bit unusual to `Clone` and receive a different type (i.e. pointer-to-value).